samples/mini-games/gradle.properties

```diff
@@ -2,4 +2,5 @@
 # kotlin.mpp.enableGranularSourceSetsMetadata=true
 # kotlin native compiler is now integrated with the gradle daemon, however, the default memory
 # settings for the daemon is not enough for kotlin native. This setting bumps the maximum memory to 3gb.
-org.gradle.jvmArgs=-Xmx3G
\ No newline at end of file
+org.gradle.jvmArgs=-Xmx3G
+godotVersion=3.2
```

Comment:
I would add a comment here stating in which file one needs to change the version as well because this is likely the place where one would change it and maybe forget or don't know about the other files where the version needs to be changed as well

Reply:
That's not anymore needed, I forgot it.